src/components/studies/scheduler/ConfigureBurstTab.tsx

```diff
@@ -129,41 +212,59 @@ const ConfigureBurstTab: FunctionComponent<ConfigureBurstTabProps> = ({
     onUpdate(schedule)
   }
 
+  const displayBurstInfoText =
+    schedule && hasBursts && (burstNumber || 0) > 0 && (burstFrequency || 0) > 0
   return (
     <div className={classes.root}>
-      <MTBHeadingH1>Multiple scheduling of the same events</MTBHeadingH1>
-      <p>
-        Ecological Momentary Assessment (EMA) provides a way to repeatedly
-        sample participant over time to better capture their ehaviors and
-        experiences in real time, also referred to as Study Bursts.
-      </p>
-      <p>
-        Example: Participants will take a session of 4 assessments everyday for
-        a week. They redo this burst every 3 months for the next two years.
+      <MTBHeadingH1 className={classes.burstDesignHeader}>
```

Comment:
The MTBHeading... controls have the style embedded in them. If it is not what you need -- pls change to the different header control. If there are additional things that you need -- just include that to the class